app/src/main/java/com/onboarding/parkingsystemjava/mvp/presenter/SetParkingPresenter.java

```diff
@@ -0,0 +1,35 @@
+package com.onboarding.parkingsystemjava.mvp.presenter;
+
+import android.util.Log;
+
+import com.onboarding.parkingsystemjava.fragment.SetParkingLotsDialog;
+import com.onboarding.parkingsystemjava.mvp.contract.SetParkingContract;
+
+public class SetParkingPresenter implements SetParkingContract.SetParkingPresenter {
+    private SetParkingContract.SetParkingView view;
+    private SetParkingLotsDialog.OnInputListener inputListener;
+    private static final String TAG = "SetParkingPresenter";
+
+    public SetParkingPresenter(SetParkingLotsDialog.OnInputListener inputListener, SetParkingContract.SetParkingView v) {
+        this.view = v;
+        this.inputListener = inputListener;
+    }
+
+    @Override
+    public void OnOkButtonPress() {
+        String lots = view.getLots();
+
+        if (!lots.equals("")) {
```

Comment:
use an EMPTY_STRING constant, do not hardcode pls.. you can create a ConstantUtils file for this kind of things

Reply:
done